src/AppBundle/Controller/PagesController.php

```diff
@@ -0,0 +1,96 @@
+<?php
+
+namespace AppBundle\Controller;
+
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Template;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Method;
+use Symfony\Bundle\FrameworkBundle\Controller\Controller;
+use Symfony\Component\HttpFoundation\Request;
+use AppBundle\Entity\Post;
+use AppBundle\Form\PostForm;
+use AppBundle\Model\PostModel;
+use AppBundle\Model\RandomDataModel;
+
+class PagesController extends Controller
+{
+    /**
+     * @Route("/", name="homepage")
+     * @Template()
+     */
+    public function indexAction()
+    {
+        $file_name = $this->container->getParameter('storage');
+        $model = new PostModel();
+        $result = $model->get($file_name);
+        $model = new RandomDataModel();
+        $images = $model->getImages(6);
+
+        return array('blogs' => $result, 'images' => $images);
+    }
+
+    /**
+     * @Route("/about", name="about")
+     * @Template()
+     */
+    public function aboutAction()
+    {
+        $model = new RandomDataModel();
+        $images = $model->getImages(4);
+
+        return array('images' => $images);
+    }
+
+    /**
+     * @Route("/contact", name="contact")
+     * @Template()
+     */
+    public function contactAction()
+    {
```

Comment:
action should was returned options for template or empty array for annotation template